refmt_test/basic.t/input.re

```diff
@@ -0,0 +1,142 @@
+/* Copyright (c) 2015-present, Facebook, Inc. All rights reserved. */
```

Comment:
There's `basic` and `basics`. In `basic` we only test types, so maybe basic-types?_¿

Didn't want to change the names since it would be easy to track down by history